tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php

```diff
@@ -1,19 +1,10 @@
-<?php  //[STAMP] 2ab9889b9623cb01b27a96cd0a920d95
```

Comment:
Shouldn't this be under gitignore?

Reply:
I'm not sure why but we have this file under version control in all our modules.

Reply:
Ok. It's not a blocker